ansible_collections/arista/avd/roles/eos_designs/docs/Fabric Variables.md

```diff
@@ -0,0 +1,36 @@
+---
+search:
+  boost: 2
+---
+
+# Fabric Variables
+
+## Evpn Settings
```

Comment:
Fix titles in table to also leverage key to display name

Reply:
Carl logged an issue on this. We will fix before releasing 4.0.